### PR TITLE
First background

### PR DESCRIPTION
**Pull request by jdtsmith**
Initial draft of background info

### CODE REVIEW

File: docs/background.rst [hunk 1]
```diff
@@ -2,4 +2,20 @@
 Background
 ##########
 
-Overall background on PAHFIT.  Motivation, philosophy, etc.
+``PAHFIT`` was developed in 2004-2006 as part of the Spitzer IRS Nearby Galaxy Survey (SINGS) to aid in the analysis of Spitzer/InfraRed Spectrograph (IRS) resolved spectral mapping of nearby galaxies, emphasizing the two low-resolution modules of IRS (Short-Low 5-15 microns and Long-Low 15-38 microns).  The model itself was designed to be as simple as possible, while still accommodating the physical variation in extragalactic MIR emission spectra.  The model combines simplified starlight, dust continuum (in the form of multi-temperature modified blackbodies), resolved dust features (e.g. PAH features), some comprised of constrained sub-features, unresolved emission lines (from ions and molecules), and screen or mixed attenuation based on a simplified extinction curve.  Due to the fitting modality,
```
Comment (karllark) [inline docs/background.rst:5]: Spitzer *Infrared* Nearby Galaxy Survey.  ;-)

**Comment by karllark** [comment]
Ready for review?  Did a quick read and it looks great!

**Comment by jdtsmith** [comment]
> Ready for review? Did a quick read and it looks great!

Yep

File: docs/background.rst [hunk 2]
```diff
@@ -2,4 +2,20 @@
 Background
 ##########
 
-Overall background on PAHFIT.  Motivation, philosophy, etc.
+``PAHFIT`` was developed in 2004-2006 as part of the Spitzer Infrared Nearby Galaxy Survey (SINGS) to aid in the analysis of Spitzer/InfraRed Spectrograph (IRS) resolved spectral mapping of nearby galaxies, emphasizing the two low-resolution modules of IRS (Short-Low 5-15 microns and Long-Low 15-38 microns).  The model itself was designed to be as simple as possible, while still accommodating the physical variation in extragalactic MIR emission spectra.  The model combines simplified starlight, dust continuum (in the form of multi-temperature modified blackbodies), resolved dust features (e.g. PAH features), some comprised of constrained sub-features, unresolved emission lines (from ions and molecules), and screen or mixed attenuation based on a simplified extinction curve.  Due to the fitting modality,
```
Comment (karllark) [inline docs/background.rst:5]: The last sentence seems to be missing an ending.

**Review by karllark — CHANGES_REQUESTED** [review]
Looks good.  Just one comment that needs addressing.

**Review by karllark — APPROVED** [review]
Looks great!

**Comment by karllark** [comment]
Ready to merge?

**Pull request merged**